lib/omlet/index.ts

```diff
@@ -0,0 +1,98 @@
+const shell = require('shelljs');
+const fs = require('fs-extra');
+const path = require('path');
+
+// name of project should match github repo name ie factiva-design-system
+const projectsToAnalyse = {
+  'factiva-design-system': {
+    'git-url': 'https://github.com/newscorp-ghfb/factiva-design-system',
+    'input-flag-path': 'src/components/**/{index, styled}.{jsx,js,ts,tsx}',
+    'ignore-flag-path': 'src/components/**/*.{stories,test}.{jsx,js,ts,tsx}',
+  },
+  'nu-sun-titan': {
+    'git-url': 'https://github.com/newsuk/nu-sun-titan',
+    packages: {
+      'nuk-ge-sun-web-component-library': {
+        'input-flag-path': 'src/components/**/*.{jsx,js,ts,tsx}',
+        'ignore-flag-path': 'src/components/**/*.{stories,test,spec}.{jsx,tsx}',
+      },
+    },
+  },
+  'ncu-newskit-render': {
+    'git-url': 'https://github.com/newscorp-ghfb/ncu-newskit-render',
+    packages: {
+      'shared-components': {
+        'input-flag-path': 'src/**/*.{jsx,js,ts,tsx}',
+        'ignore-flag-path': 'src/**/**/*.{stories,test}.{jsx,js,ts,tsx}',
+      },
+      checkout: {
+        'input-flag-path': 'src/**/**/**/*.{jsx,js,ts,tsx}',
+        'ignore-flag-path': 'src/**/*.{stories,test}.*',
+      },
+    },
+  },
+};
+
+const gitClonePromisified = project =>
```

Comment:
you don't need to specify the type here
```suggestion
const gitClone = project =>
```